### PR TITLE
Humidity test update

### PR DESCRIPTION
**Pull request by bbrk24**
Code that is currently running on for the test. Waits 24 hours, then writes temperature and humidity to the SD card.

### CODE REVIEW

**Review by CClarke513 — APPROVED** [review]
This is a-okay!

File: sketch_oct9a.ino [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#if false
+#if true
```
Comment (DarthSethious1) [inline sketch_oct9a.ino:1]: I'm confused why this was changed to 'true', and not the new file 'test_humidity'?

File: test_humidity.ino [hunk 2]
```diff
@@ -1,48 +1,47 @@
...
-  static unsigned int backoff = 100;
-
-  if (hum.getStatus() != HumiditySensor::ACTIVE) {
-    Serial.print("Not connected. Trying again in ");
-    Serial.print(backoff);
-    Serial.println("ms...");
-    delay(backoff);
-    backoff *= 2;
+  // wait a full day
+  delay(24 * 60 * 60 * 1000);
```
Comment (DarthSethious1) [inline test_humidity.ino:19]: i like just leaving this a calculation, good idea

File: test_humidity.ino [hunk 3]
```diff
@@ -1,48 +1,47 @@
...
+  while (!hum.getValues(&humidity, &temperature)) {
+  }
+
+  card.writeToCSV(fakeCoords, accel, 0, 0, humidity, temperature, gyro);
+
+  card.closeFile();
+  digitalWrite(6, LOW);
+}
+
+void loop() {
```
Comment (DarthSethious1) [inline test_humidity.ino:49]: This is fine for the current test that is running, but I would much prefer more than just one measurement. Even if it was still in 'setup' I would still take maybe 10, or 100, or something and get the average.

**Review by DarthSethious1 — CHANGES_REQUESTED** [review]
Code looks good, I would just like to see that we're taking more than one measurement for the humidity.

File: sketch_oct9a.ino (hunk 1 above)
Comment (bbrk24) [reply sketch_oct9a.ino:1]: I obviously had test_humidity true when uploading the code, but I think it makes the most sense for this file to have it on the master branch.

Comment (DarthSethious1) [reply sketch_oct9a.ino:1]: sure, i gotcha

**Pull request merged**